packages/backend/src/account/model.ts

```diff
@@ -26,4 +26,9 @@ export class Account extends BaseModel {
   public readonly balanceId!: string
   // TigerBeetle account id tracking amount sent
   public readonly sentBalanceId?: string
+
+  // TigerBeetle account id tracking an invoice's receive limit.
+  // - "credits" is the actual limit.
+  // - "debits" is the progress towards that limit.
```

Comment:
```suggestion
  // - "debits" is the actual limit.
  // - "credits" is the progress towards that limit.
```

Reply:
This comment _should_ be correct but credits and debits are swapped elsewhere in Rafiki